contracts/interfaces/IDCAStrategies.sol

```diff
@@ -0,0 +1,249 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.8.7 <0.9.0;
+
+import '@openzeppelin/contracts/token/ERC721/extensions/IERC721Enumerable.sol';
+import '@mean-finance/nft-descriptors/solidity/interfaces/IDCAHubPositionDescriptor.sol';
+
+interface IDCAStrategiesBase {
+  enum Permission {
+    INCREASE,
+    REDUCE,
+    WITHDRAW,
+    TERMINATE,
+    SYNC
+  }
+
+  struct Position {
+    address hub; // 20 bytes
+    uint80 strategyId; // 10 bytes
+    uint16 strategyVersion; // 2 bytes
+    uint256[] positions;
+  }
+
+  struct ShareOfToken {
+    address token;
+    uint80 share; // 0 < share <= 100%
+  }
+
+  struct PermissionSet {
+    address operator;
+    Permission[] permissions;
+  }
+
+  struct Strategy {
+    address owner;
+    string name;
+    ShareOfToken[] tokens;
+  }
+}
+
+interface IDCAStrategiesManagementHandler is IDCAStrategiesBase {
+  function getStrategy(uint80 _strategyId) external view returns (Strategy memory);
+
+  function getStrategyIdByName(string memory _strategyName) external view returns (uint80 _strategyId);
```

Comment:
Could you remove the underscore from these and other functions in the interface?
We usually go with the following approach:
- Interfaces have no underscore since they are only describing the parameters
- Implementations have the underscore since they might clash with other functions/variables
```suggestion
  function getStrategy(uint80 strategyId) external view returns (Strategy memory);

  function getStrategyIdByName(string memory strategyName) external view returns (uint80 strategyId);
```